api/middlewares/auth/can-read-note.middleware.js

```diff
@@ -15,7 +15,7 @@ const canReadNote = async (req, res, next) => {
 	const { data, error } = await supabase
 		.from('user_notes_with_rights')
 		.select('can_read')
-		.eq('username', req.user.username)
+		.eq('username', req.user.user_id)
```

Comment:
Are you sure you want to compare username and user_id?
Shouldn't it be `.eq('username', res.user.username)` like it was before?